src/organisation/components/pba-numbers-form/pba-numbers-form.component.html

```diff
@@ -0,0 +1,53 @@
+<app-hmcts-main-wrapper [backLink]="'/organisation'" [title]="title" [summaryErrors]="summaryErrors">
+  <div>
+    <p>We will need to verify any new PBA numbers before they can be used to pay for a case.</p>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <table class="govuk-table" *ngIf="updatePbaNumbers.hasDisplayPbaNumbers; else noPbaNumbersMessage">
+          <tbody class="govuk-table__body">
+            <tr class="govuk-table__row" *ngFor="let pba of updatePbaNumbers.displayPbaNumbers">
+              <th class="govuk-table__header" scope="row">{{pba}}</th>
+              <td class="govuk-table__cell">
+                <a class="govuk-link fright" id="remove-pba-account-number__link" [routerLink]=""
```

Comment:
`fright`? 😂

Reply:
Yes that is the name of the CSS class

Reply:
(float - right)

Reply:
Ah, that makes sense. I was just laughing at the name, not complaining. 😁